modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,115 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-aws_{context}"]
+
+= Adding an ARM64 compute machine set to your cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`). 
+* You created a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary.
+
+
+.Procedure
+
+* Create and modify a compute machine set, this will control the ARM64 compute nodes in your cluster. 
++ 
+[source,terminal]
+----
+$ oc create -f aws-arm64-machine-set-0.yaml
+----
+
++
+.Sample YAML compute machine set to deploy a ARM64 compute node
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+  name: <infrastructure_id>-aws-arm64-machine-set-0 
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+        machine.openshift.io/cluster-api-machine-role: <role> 
+        machine.openshift.io/cluster-api-machine-type: <role> 
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-<role>-<zone> 
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/<role>: "" 
+      providerSpec:
+        value:
+          ami:
+            id: ami-02a574449d4f4d280 <1>
+          apiVersion: awsproviderconfig.openshift.io/v1beta1
+          blockDevices:
+            - ebs:
+                iops: 0
+                volumeSize: 120
+                volumeType: gp2
+          credentialsSecret:
+            name: aws-cloud-credentials
+          deviceIndex: 0
+          iamInstanceProfile:
+            id: <infrastructure_id>-worker-profile 
+          instanceType: c6g.xlarge <2> 
```

Comment:
<nit>: i would use m6g in the example rather than c6g as that is the installer default